include/ts/InkAPIPrivateIOCore.h

```diff
@@ -138,64 +138,6 @@ tsapi void TSIOBufferBlockDestroy(TSIOBufferBlock blockp);
 typedef void *INKUDPPacket;
 typedef void *INKUDPacketQueue;
 typedef void *INKUDPConn;
```

Comment:
Why not also delete these three INKUDPxxx typedefs?

Reply:
I think they are being utilized